nixos/modules/programs/bash/ls-colors.nix

```
@@ -13,11 +13,17 @@ in
     programs.bash.enableLsColors = lib.mkEnableOption "extra colors in directory listings" // {
       default = true;
     };
+    programs.bash.lsColorsFile = lib.mkOption {
+      type = lib.types.str;
+      default = "";
+      example = ''''${pkgs.dircolors-solarized}/ansi-dark'';
```

This will make the example render verbatim
```suggestion
      example = lib.literalExpression "${pkgs.dircolors-solarized}/ansi-dark";
```

---

oh that nice!